docs/src/modules/utils/replaceUrl.ts

```diff
@@ -39,7 +39,7 @@ export const replaceComponentLinks = (url: string) => {
     url = url.replace(/\/components\/(.*)/, '/material-ui/$1');
   } else {
     url = url.replace(/\/components\/(.*)/, '/material-ui/react-$1');
-    if (!url.match(/\/react-(tabs|breadcrumbs)/)) {
+    if (!url.match(/\/react-(tabs|breadcrumbs|stacked-snackbars)/)) {
```

Comment:
This should not be necessary anymore. I believe that all of this logic is legacy, can be deleted.

```suggestion
    if (!url.match(/\/react-(tabs|breadcrumbs)/)) {
```

Reply:
API build script (`yarn docs:api`) is traversing this part of the code. As far as I looked, it is replacing the docs components filename path urls from plural to singular. 
Example:
`/components/checkboxes/` -> `/material-ui/react-checkbox/` (Note the `checkboxes` (plural) gets converted to singular `checkbox`). The components urls should be named singularly. Not `Checkboxes`, `Buttons`.

Whereas, this condition prevents `Tabs`, `Breadcumbs` and `Stacked snackbars` to be changed to singular word links. These should be plural.  
Example:
`/components/tabs/` -> `/material-ui/react-tabs/`
`/components/breadcumbs/` -> `/material-ui/react-breadcumbs/`

I added a test assertion for it in 481305c264f253255276006ef40a2a7f0e6102bd.

Reply:
Yes, I'm on it.